srcpkgs/python3-aionotify/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-aionotify'
+pkgname=python3-aionotify
+version=0.2.0
+revision=1
+archs=noarch
+wrksrc="aionotify-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
```

Comment:
missing depends

Reply:
Which ones ?

Reply:
Check yourself, e.g. by importing in clean masterdir.

Reply:
Yup, and it just works...

Reply:
I checked the upstream stated `requirements.txt` and I do not see what is missing here...

Reply:
`depends="python3"` is needed.